From 0e7c176b38c6ee286511e6bb7720390d31fc6233 Mon Sep 17 00:00:00 2001
From: Thomas D.
Date: Tue, 15 Mar 2016 22:21:48 +0100
Subject: [PATCH] Fix package name

The previous package name "percona-toolkit" is not a valid package name for
ExtUtils::MakeMaker:

  $ perl Makefile.PL
  Checking if your kit is complete...
  Looks good
  Warning: NAME must be a package name
  [...]

Let us use the same package name (Percona::Toolkit) various scripts in bin/
are already using.
---
 Makefile.PL | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile.PL b/Makefile.PL
index 7f46523..a9723f4 100644
--- a/Makefile.PL
+++ b/Makefile.PL
@@ -1,7 +1,7 @@
 use ExtUtils::MakeMaker;

 WriteMakefile(
-    NAME      => 'percona-toolkit',
+    NAME      => 'Percona::Toolkit',
     VERSION   => '2.2.17',
     EXE_FILES => [ <bin/*> ],
     MAN1PODS  => {
--
2.7.3
